web/src/components/tasks/business-formation/NexusFormationStepsConfiguration.ts

```diff
@@ -0,0 +1,23 @@
+import { DbaStepNames } from "@/lib/types/types";
+
+export const NexusFormationStepsConfiguration: {
+  name: DbaStepNames;
+  stepIndex: number;
+  disableStepper?: boolean;
```

Comment:
why is this false or undefined? shouldn't the other steps just have disableStepper = false?